include/cloudbreak.bash

```diff
@@ -169,7 +169,7 @@ cloudbreak-delete-certs() {
 
 cloudbreak-conf-uaa() {
 
-    env-import UAA_DEFAULT_SECRET "cbsecret2015"
+    env-import UAA_DEFAULT_SECRET "$(date | checksum sha1)"
```

Comment:
or maybe simple `$UAA_DEFAULT_USER_PW` otherwise it would changing at every run ...